modules/Activities/report_participants.php

```diff
@@ -89,7 +89,7 @@
         ->sortable(['surname', 'preferredName'])
         ->format(function ($student) use ($guid) {
             $name = Format::name('', $student['preferredName'], $student['surname'], 'Student', true);
-            return Format::link($_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID='.$student['gibbonPersonID'].'&subpage=Activities', $name);
+            return Format::link($session->get('absoluteURL').'/index.php?q=/modules/Students/student_view_details.php&gibbonPersonID='.$student['gibbonPersonID'].'&subpage=Activities', $name);
```

Comment:
This looks like it would also throw an error. Read this line and the surrounding lines and consider why this might be the case.